File: src/main/java/kitchenpos/eatinorders/tobe/infra/TobeJpaOrderTableRepository.java
package kitchenpos.eatinorders.tobe.infra;

import kitchenpos.eatinorders.tobe.domain.TobeOrderTable;
import kitchenpos.eatinorders.tobe.domain.TobeOrderTableRepository;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface TobeJpaOrderTableRepository extends TobeOrderTableRepository, JpaRepository<TobeOrderTable, UUID> {
}

File: src/main/kotlin/kitchenpos/order/tobe/eatinorder/infra/TobeJpaOrderTableRepository.kt
package kitchenpos.order.tobe.eatinorder.infra

import java.util.*
import kitchenpos.order.tobe.eatinorder.domain.ordertable.OrderTable
import kitchenpos.order.tobe.eatinorder.domain.ordertable.OrderTableRepository
import org.springframework.data.jpa.repository.JpaRepository

interface TobeJpaOrderTableRepository : OrderTableRepository, JpaRepository<OrderTable, UUID> {
}
